code/modules/mob/living/simple_animal/hostile/mimic.dm

```diff
@@ -73,8 +77,8 @@
 					"<span class='userdanger'>\The [src] knocks you down!</span>")
 
 /mob/living/simple_animal/hostile/mimic/crate/proc/trigger()
-	if(!attempt_open)
-		visible_message("<b>[src]</b> starts to move!")
+	if(!attempt_open) //Have not attempted to open

```

Comment:
comments like this are worthless, remove all comments you've added.

Reply:
Alright, what kind of comments SHOULD I be writing?